Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN useradd -u $USER_ID -d $APP_DIR appuser
 WORKDIR $APP_DIR
 COPY . $WORKDIR
 RUN chown -R $USER_ID $APP_DIR
-RUN dnf install -y java-17-openjdk-devel python3-pip
+RUN dnf install -y java-17-openjdk-devel python3-pip git
```

Comment:
Why do we need git?

Reply:
@benthomasson Its needed to build an image in development mode while we wait for the collection to be result. We can build an image by setting the branch to use 
ARG DEVEL_COLLECTION_REPO=git+https://github.com/ansible/event-driven-ansible.git

Reply:
e.g.
```
podman build --no-cache --build-arg DEVEL_COLLECTION_LIBRARY=1 --build-arg DEVEL_COLLECTION_REPO=git+https://github.com/mkanoor/event-driven-ansible.git,pg_listener -t quay.io/mkanoor/ansible-rulebook:latest .
```

Reply:
This would be addressed in https://github.com/ansible/ansible-rulebook/pull/637